ConeSearch.tex

```diff
@@ -385,97 +522,69 @@ \subsubsection{Interface}
 parameters and any custom parameters supported by the instance of the
 service.  
 
-\subsubsection{\textit{cs:}ConeSearch}
-\label{subsec:cstype}
-The \xmlel{cs:ConeSearch} type is a \xmlel{vr:Capability} sub-type that should be used
-to describe a service's support for the Simple Cone Search protocol;
-it is defined as follows:
+\subsubsection{\textit{cs:}ConeSearch} \label{subsec:cstype} The
+\xmlel{cs:ConeSearch} type is a \xmlel{vr:Capability} sub-type that
+should be used to describe a service's support for the Simple Cone
+Search protocol; it is defined as follows:
 
 % GENERATED: !schemadoc ConeSearch-v1.1.xsd ConeSearch
-\begin{generated}
-\begingroup
-      	\renewcommand*\descriptionlabel[1]{%
-      	\hbox to 5.5em{\emph{#1}\hfil}}\vspace{2ex}\noindent\textbf{\xmlel{cs:ConeSearch} Type Schema Documentation}
+\begin{generated} \begingroup \renewcommand*\descriptionlabel[1]{% \hbox
```

Comment:
If this block is generated has the generation script been changed? If not, then reformatting it will just result in churn in the document.

Reply:
fixed in commit 98611e2